Add docs 3rd try (#19)
knative/release: README.md

| v0.17   | 2020-08-18   | yanweiguo      | Harwayne        | richieescarez | -           | 2020-08-11 |
| v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott        | richieescarez | 2020-08-19  | 2020-09-22 |
| v0.19   | 2020-11-10   | julz           | n3wscott        | richieescarez | 2020-09-30  | 2020-11-03 |
| v0.20   | 2021-01-12   | nak3           | slinkydeveloper | richieescarez | 2020-11-11  | 2021-01-07 |
| v0.21   | 2021-02-23   | mattmoor       | lionelvillard   | csantanapr    | 2021-01-13  | 2021-02-16 |
| v0.22   | 2021-04-06   | markusthoemmes | evankanderson   | omerbensaadon | 2021-02-24  | 2021-03-30 |
| v0.23   | 2021-05-18   | tcnghia        | vaikas          | csantanapr    | 2021-04-07  | 2021-05-11 |
| v0.24   | 2021-06-29   | dprotaso       | matzew          | julz          | 2021-05-19  | 2021-06-22 |
| v0.25   | 2021-08-10   | JRBANCEL       | ...             | ...           | 2021-06-30  | 2021-08-03 |
| v0.26   | 2021-09-21   | ...            | ...             | ...           | 2021-08-11  | 2021-09-14 |

**NOTE:** v0.20 is moved by 3 weeks for end of year holidays

---

# Instructions

> Below you'll find the instructions to release the docs repository see [Release docs](#release-docs)

Reviewer: this comment seems unresponded to?

PR is merged, [cut a new branch](#cut-the-branch). There is no automation after
this, so you're done here.

## Release docs

> To release the documentation for the release follow [docs release process](https://knative.dev/help/maintainer/docs-release-process/)

Reviewer 2: ```suggestion
To release the documentation follow [docs release process](https://knative.dev/help/maintainer/docs-release-process/)
```

> The current process can be summarize in the following:

Reviewer 2: ```suggestion
The current process can be summarized in the following:
```

> 1. Wait for dependent repos to release (today thats /serving and /eventing)

Reviewer 3: Reasoning why I think it doesn't belong here is because that link has all the (and then some) details (how to cut a branch, etc.) and I personally found it confusing intermingled here because it's entirely different procedure currently. Client stuff that was added recently are steps that are necessary beyond the normal steps so they made sense to be in this section. WDYT?

> 1. Wait for dependent repos to release (today thats /serving and /eventing)

Reviewer 2: ```suggestion
1. Wait for dependent repos to be released (today that's /serving and /eventing)
```

> 1. Wait for dependent repos to release (today thats /serving and /eventing)

Reviewer 2: What's about splitting the document into two major sections:

* Releasing the code
* Releasing the documentation

And add a top-level TOC highlighting this difference. We then can also just say "release the docs after the code" which also automatically includes all dependencies. I think that makes sense because you are then free in the docs to reference any additional repos (beyond serving and eventing) and you don't have to take care to update this release document. I see the parallelization benefit (starting the docs release before all repositories has been published) as not big enough to introduce this additional maintenance dependency (updating this doc everytime you want to depend on a new repo in the docs)

be moved to the latest "release" of all shared dependencies prior to each
release.

The docs are released after serving and eventing are released, see [Release docs](#release-docs)

> 

Reviewer 2: See my comment above, I would suggest making this more prominent and just say: The docs should be released after the code has been released (so that the docs release team does not need to be aware of the details on which repo it depends. E.g. when including examples from r links to the released client tag or client plugins tag, then you would otherwise need to update this instruction here, which can be easily forgotten, too).

Also, as soon as you add a link to released client plugins, you would have to wait anyway until everything is released, as those plugins are the last to be released.
